models/user_test.go

```diff
@@ -74,17 +92,47 @@ func (s *DBTestSuite) TestSaveUserWithDuplicateEmail() {
 }
 
 func (s *DBTestSuite) TestSaveUserWithEmptyStringEmail() {
+	db.GetDB().Exec("DELETE FROM users")
```

Comment:
out of curiosity,  I saw that we already had `TearDown` that does the same thing here, do we really need to explicitly call this again? 🤔

Reply:
Good catch, I did not notice this and I am also wondering why it's needed.

Reply:
Because `SetupTest` always insert 1 user to database and I have more test cases that required to have user in db before run test more than not required.

I decided to delete before run the test because I want to make sure database is clean and I can assert `row in database = 0`
```go
assert.Equal(s.T(), int64(0), result.RowsAffected)
```

What do you guys think?